projects/ad5758_sdz/zed/system_constr.xdc

```diff
@@ -0,0 +1,41 @@
+
+# DAC SPI interface
+
+set_property -dict {PACKAGE_PIN E15  IOSTANDARD LVCMOS33} [get_ports spi_sclk]           ; ## FMC-LA23_P 
+set_property -dict {PACKAGE_PIN D20  IOSTANDARD LVCMOS33} [get_ports spi_sdi]            ; ## FMC-LA18_CC_P 
+set_property -dict {PACKAGE_PIN B20  IOSTANDARD LVCMOS33} [get_ports spi_sdo]            ; ## FMC-LA17_CC_N
+set_property -dict {PACKAGE_PIN B19  IOSTANDARD LVCMOS33} [get_ports spi_sync_n]         ; ## FMC-LA17_CC_P
+
+# DAC GPIO interface
+set_property -dict {PACKAGE_PIN F19  IOSTANDARD LVCMOS33} [get_ports dac_fault_n]        ; ## FMC-LA22_N 
+set_property -dict {PACKAGE_PIN E19  IOSTANDARD LVCMOS33} [get_ports dac_reset_n]        ; ## FMC-LA21_P
+set_property -dict {PACKAGE_PIN F18  IOSTANDARD LVCMOS33} [get_ports dac_ldac_n]         ; ## FMC-LA26_P 
+
+# Reconfigure the pins from Bank 34 and Bank 35 to use LVCMOS33 since VADJ must be set to 3.3V
+
+# otg
+set_property  -dict {PACKAGE_PIN  L16   IOSTANDARD LVCMOS33} [get_ports otg_vbusoc]
+
+# gpio (switches, leds and such)
```

Comment:
Why are some of the default assignments from the generic xdc redefined in this file?

Reply:
The VADJ of the Bank34 and Bank35 where the interface of the AD5758 DAC is connected must be set to 3.3V,  Some of the signals that share those banks are defined to LVCMOS25 in the generic xdc, but they actually depend on the VADJ so I changed them to LVCMOS33 here; Now I realize that setting the PACKAGE_PIN properties are redundant.

Reply:
Ok, makes sense. I overlooked that the IOSTANDARD changed.

Reply:
I added a new commit with the cleanup changes.